google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingHandler.java

```diff
@@ -310,18 +331,24 @@ public void publish(LogRecord record) {
     if (logEntry != null) {
       try {
         Iterable<LogEntry> logEntries =
-            redirectToStdout
-                ? Instrumentation.populateInstrumentationInfo(ImmutableList.of(logEntry)).y()
-                : ImmutableList.of(logEntry);
+            logTarget == LogTarget.CLOUD_LOGGING
+                ? ImmutableList.of(logEntry)
+                : Instrumentation.populateInstrumentationInfo(ImmutableList.of(logEntry)).y();
         if (autoPopulateMetadata) {
           logEntries =
               logging.populateMetadata(
                   logEntries, getMonitoredResource(), "com.google.cloud.logging", "java");
         }
-        if (redirectToStdout) {
-          logEntries.forEach(log -> System.out.println(log.toStructuredJsonString()));
-        } else {
```

Comment:
it stands out to me that this is an if else and is replaced with a switch without a default case, but 🤷

Reply:
@nickkieffer , are you using `redirectToStdout` feature?

Reply:
no, not directly anyways

Reply:
@nickkieffer , I must admit that I have some confusion while following the issue you raised, but let me try answering best based on my understanding:

1. Correct me if I wrong, you use a Python logger - based on sample code you shared. This change to print to `STDERR` was performed only in Java library and I believe we do not have such feature in Python. If problem indeed related to Python SDK, can you please raise the issue in [Python](https://github.com/googleapis/python-logging/issues) repo? Feel free to provide all details you mentioned here and also the details about environment you run in - adding @daniel-sanche for visibility.
2. This change supposed to use `CLOUD_LOGGING` by default - the `STDOUT` and `STDERR` logging needs to be explicitly enabled in code or configuration so the change should be fully backward compatible to clients which does not use the feature. If you believe this is not a case, please let us know - me and @bpcreech can take a look.

Please let me know if I understood you correctly and if I can assist you any further.

Reply:
You can disregard me on this, I think my ops team just so happened to turn off of logging agents around the same time as your release happened. Sorry about the distraction!